include/zephyr/dt-bindings/clock/stm32n6_clock.h

```diff
@@ -45,26 +45,44 @@
 #define STM32_SRC_IC18		(STM32_SRC_IC17 + 1)
 #define STM32_SRC_IC19		(STM32_SRC_IC18 + 1)
 #define STM32_SRC_IC20		(STM32_SRC_IC19 + 1)
+#define STM32_SRC_HSI_DIV	(STM32_SRC_IC20 + 1)
+#define STM32_SRC_TIMG		(STM32_SRC_HSI_DIV + 1)
+
+/** Others: Not yet supported */
+/* #define STM32_SRC_I2SCKIN	TBD */
 
 /** Bus clocks */
 #define STM32_CLOCK_BUS_AHB1	0x250
 #define STM32_CLOCK_BUS_AHB2	0x254
 #define STM32_CLOCK_BUS_AHB3	0x258
 #define STM32_CLOCK_BUS_AHB4	0x25C
 #define STM32_CLOCK_BUS_AHB5	0x260
+
 #define STM32_CLOCK_BUS_APB1	0x264
-#define STM32_CLOCK_BUS_APB1_2	0x268
 #define STM32_CLOCK_BUS_APB2	0x26C
-#define STM32_CLOCK_BUS_APB3	0x270
 #define STM32_CLOCK_BUS_APB4	0x274
-#define STM32_CLOCK_BUS_APB4_2	0x278
```

Comment:
Do not remove. Even if not used now, they could be used in the future by new peripherals.